rts/Lua/LuaConstCOB.cpp

```diff
@@ -184,8 +184,8 @@ bool LuaConstCOB::PushEntries(lua_State* L)
 	PUSH_COB(WEAPON_SPRAY);
 	PUSH_COB(WEAPON_RANGE);
 	PUSH_COB(WEAPON_PROJECTILE_SPEED);
-	PUSH_COB(MIN);
-	PUSH_COB(MAX);
+	PUSH_COB(COB_MIN);
+	PUSH_COB(COB_MAX);
```

Comment:
I think this can break existing Lua scripts because it pushes string - value pair. Perhaps replace these two calls with something like `PUSH_COB_NAMED(name, COB_....)`

Reply:
Thanks for catching, fixed :)